sdk/communication/Azure.Communication.CallAutomation/src/Models/InterruptAudioAndAnnounceResult.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using System;
+using System.Threading;
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>The result from playing audio.</summary>
+    public class InterruptAudioAndAnnounceResult
```

Comment:
Given that all of this is also a part of Play, is this necessary or can it be part of PlayResult?

Reply:
That's a good question. I can argue that it confuses less people to have a separated result object. I agree as well that this is kind of PlayResult. 
I can ask directly to azure board on the review and see what they think about it.